index.js

```diff
@@ -1,16 +1,135 @@
-const http = require('http');
+var http = require("http");
+var express = require('express');
+var nlp = require('compromise');
+var app = express();
+var bodyParser = require('body-parser');
+var urlencodedParser = bodyParser.urlencoded({ extended: true });
 
-//const hostname = '127.0.0.1';
-//const port = 3000;
+var PORT = process.env.PORT || 5000;
 
-const PORT = process.env.PORT || 5000;
-
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('The {"Location": "Philadelphia"} {"Organization": "Flyers"} are the best hockey team in the {"Organization": "NHL"}, and {"Person": "Claude Giroux"} should have won {"Organization": "MVP"}.\n');
+// Running Server Details.
+var server = app.listen(PORT, function () {
+  var host = server.address().address
+  var port = server.address().port
+  console.log("Example app listening at %s:%s", host, port)
 });
-
-server.listen(PORT, () => {
-  console.log(`Server running ${PORT}/`);
+ 
+ 
+app.get('/', function (req, res) {
+  var html=printHtmlForm('','LIST');
+  res.send(html);
 });
+ 
+app.post('/tagged', urlencodedParser, function (req, res){
+  var input = req.body.inputString;
+  var outputFormat = req.body.outputFormat;
+  var reply=printHtmlForm(input, outputFormat);
+  reply += "<br/><b>The tagged input is shown below:</b><br/>";
+  var tags = tagInput(input, outputFormat);
+  reply += tags;
+  res.send(reply);
+ });
+
+ function tagInput(data, format) {
+	if (format === 'LIST') {
+		return tagList(data);
+	} else if (format === 'JSON') {
+		return tagJson(data);
+	} else if (format === 'XML') {
+		return tagXml(data);
+	} else {
+		return "Error, unknown output format."
+	}
+ }
+
+function tagList(data) {
+        var inputString = data;
+        var tags = "<textarea rows='10' cols='80' readonly='true'>People: &#13;&#10;";
+        var doc = nlp(inputString);
+        tags = tags + doc.people().out();
+        tags = tags + "&#13;&#10;&#13;&#10;Places: &#13;&#10;";
```

Comment:
What is this? Up to you, but I personally would either use a variable with a useful name to describe what each special char is, or a comment to explain what you're doing with the ambersand commands for special characters.